VisualStudio.gitignore

```diff
@@ -183,6 +183,7 @@ AppPackages/
 BundleArtifacts/
 Package.StoreAssociation.xml
 _pkginfo.txt
+.appx
```

Comment:
Cool. Looks like we need `*.appx` then.

Given we can't rely on `AppPackages/` here we need to figure out what to do about that rule - are there any other files that are output as part of this process that need to be excluded?